### PR TITLE
Administration panel: Create panel admin view home#index

### LINKED ISSUE #17: Administration panel: Create panel admin view home#index
Include:
- [x] link to new shipment page
- [x] link to new user page
- [x] link to reports page

Add routes.

### PR DESCRIPTION
**Pull request by diegotc86**
Close #17 
Links to sales, new user and reports created (temporal views created also)
Searching a valid tracking_id send you to a form to update the delivered date which send the corresponding emails.

### CODE REVIEW

File: app/controllers/admin/shipments_controller.rb [hunk 1]
```diff
@@ -2,15 +2,18 @@ module Admin
...
     end
 
     def update
       shipment = Shipment.find(params[:id])
       if shipment.update(shipment_params)
         ShipmentMailer.with(shipment: shipment).shipment_delivered.deliver_now
-        redirect_to(root_path)
+        redirect_to(admin_root_path)
       else
         render :edit
```
Comment (Linzeur) [inline app/controllers/admin/shipments_controller.rb]: I see that you delete the edit action, then you should change this line

File: app/controllers/shipments_controller.rb [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
   
-  def show
+  def search
     @shipment = Shipment.find_by_tracking_id(params[:tracking_id])
 
     unless @shipment
       flash[:notice] = "The shipment doesn't exist"
       redirect_to root_path
+    else
+      render :show
```
Comment (Linzeur) [inline app/controllers/shipments_controller.rb]: Here you should  replace the show by search

File: config/routes.rb [hunk 3]
```diff
@@ -15,7 +15,9 @@
   namespace :admin do
     root 'home#index'
     get  'sales', to: 'sales#report'
-    resources :shipments, only: [:edit, :update]
+    get 'mark_delivered', to: 'shipments#delivered'
```
Comment (Linzeur) [inline config/routes.rb]: Thanks for fix that

File: app/controllers/admin/shipments_controller.rb (hunk 1 above)
Comment (diegotc86) [reply app/controllers/admin/shipments_controller.rb]: Thanks, it has been corrected

File: app/controllers/shipments_controller.rb (hunk 2 above)
Comment (diegotc86) [reply app/controllers/shipments_controller.rb]: In this case, the view name is "show"

Comment (Linzeur) [reply app/controllers/shipments_controller.rb]: Oh you're right!!

**Pull request merged**